### PR TITLE
Bug 1190713 - [Settings] Port a few pack-switches to use web components

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1190713

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=41a00a9d7b3d095d7ace4ac26434e6c0eaa41ef8

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c0bb03ee18dbc815cc09b2a71451ae368358a1ca

File: apps/settings/elements/wifi.html [hunk 1]
```diff
@@ -45,10 +44,9 @@ <h2 data-l10n-id="advancedSettings"></h2>
       </header>
       <ul>
         <li class="hint">
-          <label class="pack-switch">
-            <input type="checkbox" name="wifi.sleepMode">
-            <span data-l10n-id="wifi-sleep"></span>
-          </label>
+          <gaia-switch name="wifi.sleepMode">
+            <label data-l10n-id="wifi-sleep"></label>
+          </gaia-switch>
```
Comment (gasolin) [inline apps/settings/elements/wifi.html]: click the description part wont toggle the switch, seem like related to bug 1190548

File: apps/settings/elements/search.html [hunk 2]
```diff
@@ -7,25 +7,19 @@ <h1 data-l10n-id="search-header"></h1>
...
             </select>
           </div>
         </li>
         <li>
-          <label class="pack-switch">
-            <input type="checkbox" name="search.suggestions.enabled">
-            <span data-l10n-id="search-suggestions">Search Suggestions</span>
+          <gaia-switch name="search.suggestions.enabled">
+            <gaia-switch data-l10n-id="search-suggestions"></gaia-switch>
           </label>
```
Comment (gasolin) [inline apps/settings/elements/search.html]: should remove the label tag, or the switch looks weird in search panel.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b2a2f9e659d4c432c08223d142d036e5fbae5845

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6b83f5afb3cebf39bee6d98b16885294dbb11c0b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=70e8965124352142fdab909ac3954bcee896c8c7

**Pull request merged**